i18n/fr.json

```diff
@@ -7,8 +7,8 @@
     "en": "fr",
     "Paste does not exist, has expired or has been deleted.":
         "Le paste n'existe pas, a expiré, ou a été supprimé.",
-    "%s requires php 5.3.0 or above to work. Sorry.":
-        "Désolé, %s nécessite php 5.3.0 ou supérieur pour fonctionner.",
+    "%s requires php 5.4.0 or above to work. Sorry.":
+        "Désolé, %s nécessite php 5.4.0 ou supérieur pour fonctionner.",
```

Comment:
As stated in https://github.com/PrivateBin/PrivateBin/issues/201#issuecomment-288960981, please replace the version number with `%s`.